### ISSUE TITLE
[rush] `ERROR: File does not exist: ... install-run-rush-pnpm.js` when upgrading to 5.88.1

### ISSUE THREAD

**Issue by joelrbrandt** [microsoft/rushstack#3905]
## Summary

I modified my `rush.json` to use rush version 5.88.1 (going from 5.86.0). Then I ran `rush update`. I got the following errors:

```
ERROR: File does not exist: /my/home/dir/.nvm/versions/node/v16.13.0/lib/node_modules/@microsoft/rush/node_modules/@microsoft/rush-lib/dist/scripts/install-run-rush-pnpm.js
File does not exist: /my/home/dir/.nvm/versions/node/v16.13.0/lib/node_modules/@microsoft/rush/node_modules/@microsoft/rush-lib/dist/scripts/install-run-rush-pnpm.js
ENOENT: no such file or directory, open '/my/home/dir/.nvm/versions/node/v16.13.0/lib/node_modules/@microsoft/rush/node_modules/@microsoft/rush-lib/dist/scripts/install-run-rush-pnpm.js'
```

Upgrading from `5.86.0` to `5.88.0` works fine. However, we can't use `5.88.0` due to #3889 

Indeed, looking at the built product for `@microsoft/rush-lib@5.88.1` (using `npm install @microsoft/rush-lib@5.88.1`) suggests those files are in the wrong place: they are in `.../@microsoft/rush-lib/dist/undefined/` instead of `.../@microsoft/rush-lib/dist/scripts/`

## Repro steps

1. Have a repo at rush version < 5.88.1
2. Modify rush.json to use rush version 5.88.1
3. Run `rush update`

  **Expected result:** 

Scripts updated.

  **Actual result:** 

Error, as above.

## Standard questions

Please answer these questions to help us investigate your issue more quickly:

| Question | Answer |
| -------- | -------- |
| `@microsoft/rush` globally installed version? | 5.88.1 |
| `rushVersion` from rush.json? | 5.88.1 |
| `useWorkspaces` from rush.json? | true |
| Operating system? | Mac |
| Would you consider contributing a PR? | Yes, but not for this issue |
| Node.js version (`node -v`)? | 16.13.0 (also repros with 18.13.0) |

**Comment by andywgarcia** [comment]
Same problem here and same fix as https://github.com/microsoft/rushstack/issues/3905#issuecomment-1397034060

**Comment by ymei0072** [comment]
Ran into the same issue and had to manually rename a folder (`undefined` to `scripts`) to be able to rush update

**Comment by Twin-2** [comment]
Also had this same issue with the same fix working.

**Comment by iclanton** [comment]
Sorry everyone - this has been fixed in Rush 5.88.2

**Comment by joelrbrandt** [comment]
@iclanton thanks for coming through with a fix so quickly, and on a weekend no less. ...and, given everything that has been going on in tech over the last week.

We appreciate all you and your team do -- Rush is a huge benefit to our organization.
